pages/bitmark-references/bitmark-sdk/websocket.md

```diff
@@ -19,7 +19,7 @@ Bitmark SDK offers Web Socket event triggering to help the application can be no
   + `txId`: the corresponding transaction id
   + `presence`: indicate that the bitmark is owned or not by the corresponding `Account`
 - New transfer offer: This event will be sent when users receive new transfer offer (for 2 signatures transfer) from someone.
-  + `bitmarkId`: the id of bitmark is comming
+  + `bitmarkId`: the id of bitmark is coming
```

Comment:
I do know know what this means

Reply:
@jamieabc: Could you check and correct the definition? I think you wrote the websocket.md